core/domain/user_services_test.py

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+from contextlib import nested
```

Comment:
Can you give them names? https://github.com/oppia/oppia/blob/db43cf648914e8377622605df6b0833215d12a5d/core/platform/search/gae_search_services_test.py#L156